CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKHitTester.java

```diff
@@ -434,7 +442,10 @@ public int DisplayUpdate() {
             }
         }
         
-        needsDisplayUpdate = false;
+        // If there are still requests that need to be processed then continue updating
+        // next frame otherwise we can cause a deadlock with a request response
+        // waiting (blocking the VisualManager) forever for a response.
```

Comment:
I feel like the right solution is move the hit testing code (not the rendering of the picking buffer) higher and not be part of the render loop.

Reply:
Maybe we change the visual manager to always pump the hit test code.  The hit tester can request a redraw if it doesn't have a current pick buffer.